Makefile

```diff
@@ -213,6 +214,68 @@ dist/agentctl-freebsd-amd64:
 dist/agentctl-windows-amd64.exe:
 	@CGO_ENABLED=1 GOOS=windows GOARCH=amd64; $(seego) build $(CGO_FLAGS) -o $@ ./cmd/agentctl
 
+PACKAGE_IN_CONTAINER := true
+PACKAGE_VERSION := $(shell ./tools/package-version 2>/dev/null)
+# The number of times this version of the software was released, starting with 1 for the first release.
+PACKAGE_RELEASE := 1
+PACKAGE_IMAGE ?= $(IMAGE_PREFIX)/fpm
```

Comment:
Can these be moved to the top of the file with the other options? `FPM_OPTS` and `FPM_ARGS` can stay here since they're very localized

Reply:
Sure thing!